### ISSUE TITLE
Documentation not following HAL format

### ISSUE THREAD

**Issue by zmcode** [springfox/springfox#1927]
Hi everyone,
I am using springfox 2.7.0 together with spring boot data rest. 
In the documentation produced by springfox the response has the following format:
```
{ 
  "content": [ <list of resources> ]
  "links": []
}
```
whereas the actual returned response has the following one:
```
{
  "_embedded": { "collectionName": [ <list of resources> ] }
  "_links": {}
}
```
Is this a known issue or am I doing something wrong in configuring either springfox or spring boot data rest?

**Comment by dilipkrish** [comment]
@zmcode that is currently happening because those are json representations of the resource. Looking to see how to make this better.

**Comment by zmcode** [comment]
@dilipkrish ok, thank you for your answer!

**Comment by ccschneidr** [comment]
Any news on this regard?

**Comment by jeeftor** [comment]
Anybody come up with a workaround for this?  Could you make a stand-in alternate type rule or something to fix this?

**Comment by jeeftor** [comment]
@dilipkrish - will this fix be included in version 2.8?
